libs/filamat/src/Enums.cpp

```diff
@@ -47,6 +47,8 @@ std::unordered_map<std::string, Property> Enums::mStringToProperty = {
         { "ior",                 Property::IOR },
         { "microThickness",      Property::MICRO_THICKNESS },
         { "bentNormal",          Property::BENT_NORMAL },
+        {"specularFactor",       Property::SPECULAR_FACTOR },
+        {"specularColorFactor",  Property::SPECULAR_COLOR_FACTOR }
```

Comment:
Just call it `specularColor`, we don't use glTF naming conventions in our core material models.

Reply:
Cannot. The reason call this is because SpecularColor is already used above by the specularRoughness extension.

Reply:
The properties defined in Filament are not glTF specific. The glTF *material* can combine the properties into one or use one over the other.

Reply:
seems like `reflectance` is a good replace of 0.04 since it is default to 0.04 and calculated correctly in IOR case.